switchapi/CMakeLists.txt

```diff
@@ -60,7 +60,6 @@ add_library(switchapi_o OBJECT
 )
 
 target_include_directories(switchapi_o PRIVATE
-    ${OVS_INSTALL_PREFIX}/include
     ${SDE_INSTALL_PREFIX}/include
```

Comment:
We're going to need to revisit this later. It embeds the assumption that the utility libraries will be provided by the target driver. This may be true now, but it won't be true in the future.